src/bundle/Resources/public/js/scripts/admin.limitation.pick.js

```diff
@@ -91,8 +91,13 @@
         selectedItems.forEach((location) => {
             const locationId = location.id;
             const container = doc.createElement('ul');
-            const renderedItem = tagTemplate.replace('{{ location_id }}', locationId);
 
+            container.insertAdjacentHTML('beforeend', tagTemplate);
+
+            const tagTemplateUnescaped = container.innerHTML;
```

Comment:
I haven't found a nice fix for the double escape of the attribute in a twig, so I unescape the template here. And only after that I use it.